### ISSUE TITLE
azure doesn't get logs from running cluster (in log_collector)

### ISSUE THREAD

**Issue by fruch** [scylladb/scylla-cluster-tests#4821]
```python
    def get_running_cluster_sets(self, backend):
        if backend in ("aws", "aws-siren", "k8s-eks"):
            self.get_aws_instances_by_testid()
        elif backend in ("gce", "gce-siren", "k8s-gke"):
            self.get_gce_instances_by_testid()
        elif backend == 'docker':
            self.get_docker_instances_by_testid()
        else:
            LOGGER.debug("get_running_cluster_sets: unknown backend type: %s", backend)
```

**Comment by fgelcer** [comment]
@soyacz , the fix for it will come as part of https://trello.com/c/6VlLyPLb ?

do you have an ETA for it?

**Comment by soyacz** [comment]
> @soyacz , the fix for it will come as part of https://trello.com/c/6VlLyPLb ?
> 
> do you have an ETA for it?

Yes, it's the same.
Should be fixed in this sprint.

**Comment by fruch** [comment]
Fixed by:  #4839
